tools/SchemaManager.Core/SqlSchemaManager.cs

```diff
@@ -304,5 +303,12 @@ private async Task ValidateInstancesVersionAsync(int version, CancellationToken
                 throw new SchemaManagerException(string.Format(CultureInfo.InvariantCulture, Resources.InvalidVersionMessage, version));
             }
         }
+
+        public async Task<int> GetLatestSchema(CancellationToken cancellationToken = default)
```

Comment:
Where do we intend to use it?

Reply:
I'm going to use this in health-paas once this pr gets checked in. I have it in a branch but cant make a pr until I get the latest nugets from here.